domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/BikesAllowedStatus.java

```diff
@@ -0,0 +1,22 @@
+package org.mobilitydata.gtfsvalidator.domain.entity;
```

Comment:
Seems like these GTFS entities should be in a different package than the validator domain entities - maybe `org.mobilitydata.gtfsvalidator.domain.entity.gtfs`?

Reply:
That makes sense, will add the package

Reply:
Created #68 so we'll do it after merging our entities to master to avoid potential git troubles creating the same package in all branches.